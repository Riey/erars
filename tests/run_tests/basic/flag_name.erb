@SYSTEM_TITLE
TFLAG:Info표시중 = 1
PRINTFORML %FLAGNAME:1% %ITEMNAME:1%
PRINTFORML {ITEMPRICE:☆기교Lv} %TFLAG:Info표시중%
