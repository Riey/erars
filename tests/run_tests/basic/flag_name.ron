[Print("플래그 아이템")]
