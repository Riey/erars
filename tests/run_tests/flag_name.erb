@SYSTEM_TITLE
PRINTFORM %FLAGNAME:1% %ITEMNAME:1%
