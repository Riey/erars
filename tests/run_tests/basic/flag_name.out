플래그 아이템